practice/e-commerce-website/src/components/Button/Button.tsx

```diff
@@ -1,3 +1,6 @@
+/* Import constants */
+import { BUTTON_VARIANTS } from '../../constants/constants';
```

Comment:
rename constants.ts to index.ts and you can import { BUTTON_VARIANTS, COMPONENT_SIZES } from '../../constants'?